install_analysistoolsts.sh

```diff
@@ -41,20 +43,22 @@ do
     if [ -d "${dir}" ]; then
         echo "pulling changes ..."
         # IF THE REPO ALREADY EXISTS...
-        pushd "${dir}" || exit
+        pushd "${dir}"
+
+        CURRENT_BRANCH=$(git branch --show-current)
 
         # PULL CHANGES
         git fetch
         git reset --hard HEAD
-        git merge origin/"$CURRENT_BRANCH"
-        popd || exit
+        git merge origin/"${CURRENT_BRANCH}"
```

Comment:
`${CURRENT_BRANCH}` didn't look like it was set anywhere beforehand. Not sure what the expected value was, but assumed it was to just get the current branch name when we moved into the folder.